packages/algob/sample-project/ts/algob.config.ts

```diff
@@ -0,0 +1,89 @@
+// NOTE: below we provide some example accounts.
+// DON'T this account in any working environment because everyone can check it and use
+// the private keys (this accounts are visible to everyone).
+
+// NOTE: to be able to execute transactions, you need to use an active account with
+// a sufficient ALGO balance.
+
+/**
+	 Check our /docs/algob-config.md documentation (https://algobuilder.dev/guide/algob-config.html) for more configuration options and ways how to
+	load a private keys:
+	+ using mnemonic
+	+ using binary secret key
+	+ using KMD daemon
+	+ loading from a file
+	+ loading from an environment variable
+	+ ...
+*/
+
+// ## ACCOUNTS USING mnemonic ##
+const { mkAccounts, algodCredentialsFromEnv } = require("@algo-builder/algob");
+let accounts = mkAccounts([
+	{
+		// This account is created using `make setup-master-account` command from our
+		// `/infrastructure` directory. It already has many ALGOs
+		name: "master",
+		addr: "WWYNX3TKQYVEREVSW6QQP3SXSFOCE3SKUSEIVJ7YAGUPEACNI5UGI4DZCE",
```

Comment:
I think this is already somewhere in the project maybe we can move it to a separate file an export it so we have access to it from the rest of the project instead of copy pasting this every time?

Reply:
We can't do that. On running `yarn run algob . --typescript`, `sample-project` folder content is copied as it is. Whatever the files present inside it will be used to initialize the project via `algob`.

Reply:
can't we do import {accountConfig} or sth like that?

Reply:
No, we can't do that. `sample-project` will be copied as it is but not the files outside this project folder.